python/hawkey/__init__.py

```diff
@@ -201,6 +222,29 @@ def _has_just_name(self):
             not self.release and not self.arch
 
 
+class ModuleForm(_hawkey.ModuleForm):
```

Comment:
@mhatina The functionality of that class should be replaced by C code. In future there should be no such a layer.